dart/test/initialization_test.dart

```diff
@@ -0,0 +1,37 @@
+@TestOn('vm')
+
+import 'package:sentry/sentry.dart';
+import 'package:test/test.dart';
+
+import 'mocks.dart';
+
+// Tests for the following issue
+// https://github.com/getsentry/sentry-dart/issues/508
+// There are no asserts, test are succesfull if no exceptions are thrown.
+void main() {
```

Comment:
maybe set do a `setDown()`, nto sure if its called like this but the opposite of `setUp`, to call `Sentry.close` as well, so every test star in a clean state